modules/ROOT/pages/cassandra/cassandra-connector-xml-maven.adoc

```diff
@@ -0,0 +1,74 @@
+= Cassandra XML and Maven Support - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+If you manually code a Mule application in XML either from the Studio XML editor or standalone, to get access to a connector, you must:
+
+* Add the namespace for the connector to your XML flow.
+
+* Add dependency information to your Maven `pom.xml` file.
+
+In the Studio visual editor, when you drag a connector from the palette to the canvas, Studio automatically populates the XML code with the connector namespace and schema location.
+
+
+== Add a Namespace for the Connector
+If you manually code the Mule application in Studio’s XML editor or another text editor, paste the following into the header of your configuration XML, inside the `<mule>` tag:
+
+[source,xml,linenums]
```

Comment:
There is a space in front of [source,xml,linenums]--remove the extra space so this doesn't show up in the published content.